README.md

```diff
@@ -291,13 +301,13 @@ class IterableWeakMap {
   }
 
   *keys() {
-    for (const [key, value] of this) {
+    for (const { 0: key } of this) {
       yield key;
     }
   }
 
   *values() {
-    for (const [key, value] of this) {
+    for (const { 1: value } of this) {
       yield value;
     }
   }
```

Comment:
Using object destructuring doesn’t require an inner iteration of the `key`/`value` pair.